ee/app/livechat-enterprise/server/hooks/addDepartmentAncestors.js

```diff
@@ -0,0 +1,20 @@
+import { callbacks } from '../../../../../app/callbacks';
+import LivechatRooms from '../../../../../app/models/server/models/LivechatRooms';
+import { LivechatDepartment } from '../../../../../app/models/server/models/LivechatDepartment';
+
+callbacks.add('livechat.newRoom', (room) => {
+	if (!room.departmentId) {
+		return room;
+	}
+
+	const department = LivechatDepartment.findOneById(room.departmentId, { fields: { ancestors: 1 } });
```

Comment:
I'm getting the following error at this point:

![Screen Shot 2020-03-18 at 15 06 39](https://user-images.githubusercontent.com/59577424/76992624-49726c80-692a-11ea-8971-7b71bebf386c.png)